src/cobra/io/web/load.py

```diff
@@ -0,0 +1,188 @@
+"""Provide a function ``load_model`` to access remote model repositories."""
+
+
+import gzip
+import logging
+from typing import TYPE_CHECKING, Iterable
+
+import diskcache
+import httpx
+import libsbml
+
+from ...core import Configuration
+from ..sbml import _sbml_to_model
+from .abstract_model_repository import AbstractModelRepository
+from .bigg_models_repository import BiGGModels
+from .biomodels_repository import BioModels
+
+
+if TYPE_CHECKING:
+    from cobra.core import Model
+
+
+logger = logging.getLogger(__name__)
+configuration = Configuration()
+
+
+def load_model(
+    model_id: str,
+    repositories: Iterable[AbstractModelRepository] = (BiGGModels(), BioModels()),
+    cache: bool = True,
+) -> "Model":
```

Comment:
Should probably be `cobra.core.Model`

Reply:
Since it's imported as follows, I think it's fine.

```py
if TYPE_CHECKING:
    from cobra.core import Model
```